toolkit/imageconfigs/scripts/linuxguard/selinux-ci-config.sh

```diff
@@ -81,6 +84,7 @@ ENABLED_MODULES: set[str] = {
     "sysnetwork",
     "systemd",
     "tpm2",
+    "trident",
```

Comment:
Will this work without the Trident rpm being installed?

Reply:
IC seems to log a benign warning if the module isn't found https://dev.azure.com/mariner-org/mariner/_build/results?buildId=879297&view=logs&j=bc1e2b45-0f64-5d35-f6bc-df79a3ea66d6&t=bf9cac8b-068f-500b-4bfa-9f9aaa4d7ea5&l=9669.

If we remove this we may also have to change the selinux-ci-policy spec to mirror the change https://dev.azure.com/mariner-org/mariner/_git/steamboat/commit/69f9bae462599546eca1085fc38dfd480c9eed4b?refName=refs/heads/main&_a=contents